src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala

```diff
@@ -63,6 +63,7 @@ object MockWorkspaceServer {
     useCallCache = Option(randomBoolean()),
     deleteIntermediateOutputFiles = Option(randomBoolean()),
     useReferenceDisks = Option(randomBoolean()),
+    memoryRetryMultiplier = Option(randomDouble()),
```

Comment:
Random seems unnecessary? Could throw off someone who chooses to assert against this response in the future.